src/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ if( MBUS_FOUND )
     target_link_libraries(vzlogger ${MBUS_LIBRARY})
 endif( MBUS_FOUND )
 target_link_libraries(vzlogger ${OCR_LIBRARIES})
+target_link_libraries(vzlogger modbus)
```

Comment:
Different from mbus_library?

Reply:
Yes, modbus and mbus have little in common, except the m and the bus ;)